OMCompiler/Compiler/NBackEnd/Modules/2_Pre/NBAlias.mo

```diff
@@ -887,7 +887,7 @@ protected
       fixed_val := true;
```

Comment:
what are you using this value for? i don't see any place it is used, did you forget something here?

Reply:
You are right, i do not use this value. I will remove it in the next commit.